src/core/APIBox/apiSource.js

```diff
@@ -0,0 +1,4 @@
+export const apiKey = "d3f19b5007aaab7cb579f83b9a664dec";
+export const apiBaseUrl = "https://www.themoviedb.org/";
```

Comment:
here it should be this -> "https://api.themoviedb.org/3" you forgot to type 3 after .org/

Reply:
That's right Adrian, thanks for finding the error!